frontend/.eslintrc

```diff
@@ -1,6 +1,8 @@
 {
   "root": true,
   "env": {
+    "cypress/globals": true,
+    "node": true,
```

Comment:
You should have a separate `.eslintrc` just for the tests and not put this in the root `.eslintrc`.

Reply:
yes, fixed, see [comment](https://github.com/openshift/console/pull/4831#issuecomment-611705992)